### DISCUSSION TITLE
Disable Page Publishing Patch

### DISCUSSION THREAD

**Discussion by BillyShelton** [2sic/2sxc#2505] [Q&A]
I am having an issue with 2Sxc not publishing content and I suspect applying the [Disable Page Publishing Patch](https://github.com/2sic/2sxc/tree/9ba57100fa532aef41ec96a9bb0f347ac89aeb73/Src/Patches) might resolve the issue.  Is there any instructions on applying this patch and is it valid for the current LTS version of 2Sxc?

**Comment by iJungleboy** [comment]
What is your background? are you using Evoq or page publihsing workflow? otherwise this doesn't seem applicable.

**Reply by BillyShelton** [reply]
So we have a site that was converted from Evoq to community a few years ago.  When I install 2Sxc on the site and add content, the content shows for admins, but only "demo content" shows publicly.  If I change the Status when editing the content from "Show" to Draft or Hide, I cannot change it back to show.  Additionally, if I change that, then on the content menu there is a "eye" icon with a line through it that says unpublished, but will not let me click it to publish it.

**Reply by iJungleboy** [reply]
I can't say much for this scenario. Since it started as an Evoq, there may still be some setting affecting this. 

I guess you can only try the patch and see what happens, but it could also be that nothing happens. 

I suggest you also check the insights of the API call doing save or publish, you will probably see what happens and why.

**Reply by BillyShelton** [reply]
Hey Daniel, Thanks for the response. I looked at the insight logs and do not see anything obvious, but to be honest, most of it is pretty cryptic.  This is my first attempt at using 2Sxc. I did check when saving changes to the content and the post is successful.  If I wanted to give the patch a try, how would I do that? Or if you feel that is a waste of time, do you have any other thoughts as to what I should be looking at?
